SPACING.md

```diff
@@ -0,0 +1,335 @@
+# Spacing Style Guide
+When and where to use spaces and blank lines.
+
+### Rational
+(skip if bored)
+
+Humans primarily process and _make sense_ of data when consumed visually. Consistent spacing - and format in general - drastically lowers the cognitive load when reading code and decreases the time needed to understand it. We should consent to a single fixed style of formatting Swift files as part of a journey to better and more readable code. Hence, please enforce these guidelines when reviewing pull requests.
+
+__Note:__ All of the following suggestions are general guidelines. They should be subject to discussions and may be ignored if a convincing reason exists.
+
+---
+
+### Closing Scopes
+Simply __never__ include any whitespace before closing a scope and always __one__ blank line afterwards.
+```swift
+        return x
+    }
+}
+
+func nextFunc() {}
+```
```

Comment:
I think this is not a very good example, as I would require a comment for this case – why is the function empty.
E.g. 

```swift 
func nextFunc() {
  // TODO: implement nextFunc()
}

func nextFunc() {
  // nothing to do for now
}

func nextFunc() {
  // override in subclasses
}


```